Makefile

```diff
@@ -579,6 +587,12 @@ $(KPROMO):
 $(GOLANGCI_LINT): # building golanci-lint from source is not recommended, so we are using the install script
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(TOOLS_BIN_DIR) $(GOLANGCI_LINT_VER)
 
+$(CLUSTERCTL):
```

Comment:
initially additional tools install script was in the separate hack folder, don't we want to continue to do that ?

Reply:
I think your comment is about where the `clusterctl` executable is stored; it's at `hack/tools/bin/clusterctl`.

Please see https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/pull/349/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R130

Reply:
I've followed the same convention that's used to install `golangci-lint`.

Reply:
i was more thinking about other tools installed with script like that https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/blob/main/hack/install-go.sh
but looks like we already have two different method to install dependency 

need to fix it later

Reply:
You're right, there's a separate convention for tools we build locally using go.

I wanted to build clusterctl locally, but this requires a workaround to a known issue. I noted this in https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/pull/349/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R591-R592